R/pkg/R/schema.R

```diff
@@ -68,18 +73,10 @@ structType.structField <- function(x, ...) {
   structType(stObj)
 }
 
-#' Create \code{structType} for a given DDL-formatted string.
-#'
-#' Create \code{structType} for a given DDL-formatted string, which is a comma separated list
-#' of field definitions, e.g., a INT, b STRING.
```

Comment:
I think we should consider keeping `which is a comma separated list of field definitions, e.g., a INT, b STRING.`
actually, I'd suggest putting the string in quotes, like `"a INT, b STRING"`